src/main.py

```diff
@@ -1,8 +1,69 @@
-from fastapi import FastAPI
+import uuid
+
+from fastapi import FastAPI, HTTPException
+from src.schemas import ProjectDetails, Project
 
 app = FastAPI()
+projects: list[ProjectDetails] = []
+
+
+def find_project(project_id: uuid.UUID) -> ProjectDetails:
```

Comment:
Better name it `get_project`. Find usually means filtering, here you retrieve the specific project